lib/perl/Genome/Model/ReferenceSequence/Command/FindPotentialMatches.pm

```diff
@@ -0,0 +1,186 @@
+package Genome::Model::ReferenceSequence::Command::FindPotentialMatches;
+
+use strict;
+use warnings;
+
+use Genome;
+use Lingua::EN::Inflect;
+use UR::Util;
+
+class Genome::Model::ReferenceSequence::Command::FindPotentialMatches {
+    is => 'Command::V2',
+    has => [
+        query_file => {
+            is => 'FilePath',
+            doc => 'any TSV where the first column contains chromosome names',
+        },
+    ],
+    has_optional => [
+        taxon => {
+            is => 'Genome::Taxon',
+            doc => 'limit search to references matching this taxon',
+        },
+        include_subsets => {
+            is => 'Boolean',
+            doc => 'include references that contain a subset of the chromosome names in the query',
+            default_value => 0,
+        },
+        include_supersets => {
+            is => 'Boolean',
+            doc => 'include references that contain a superset of the chromosome names in the query',
+            default_value => 1,
+        },
+        include_partial_matches => {
+            is => 'Boolean',
+            doc => 'include references which contain any of the chromosome names in the query',
+            default_value => 0,
+        },
+    ],
+    has_transient_optional_output => [
+        exact_matches => {
+            is => 'ARRAY',
+            doc => 'Exact matches that were found',
+        },
+        supersets => {
+            is => 'ARRAY',
+            doc => 'Supersets that were found (or undef if --include-supersets was not used).',
+        },
+        subsets => {
+            is => 'ARRAY',
+            doc => 'Subsets that were found (or undef if --include-subsets was not used).',
+        },
+        partial_matches => {
+            is => 'ARRAY',
+            doc => 'Parial matches that were found (or undef if --include-partial-matches was not used).',
+        },
+    ],
+    doc => 'find reference sequences whose chromosome names match a file',
+};
+
+sub help_detail {
+    return <<EOHELP;
+This command searches through all reference sequences to find ones with chromosome names that match those present in a file.  This does not guarantee that the file actually comes from the references that match--only that the chromosome names are the same.  (For example, 1-22,X,Y will find multiple reference versions of the human reference which are supersets, but it cannot distinguish between them.)
+EOHELP
+}
+
+sub execute {
+    my $self = shift;
+
+    my $query_chromosomes = $self->_query_chromosomes;
+
+    my $reference_iterator = $self->_reference_iterator;
+
+    my @matches;
+    my @supersets;
+    my @subsets;
+    my @partials;
+    while (my $reference = $reference_iterator->next) {
+        my $seqdict_path = File::Spec->join($reference->data_directory, 'seqdict', 'seqdict.sam');
+        next unless(-e $seqdict_path); #don't try to generate this now
```

Comment:
I'm assuming that `$reference->chromosome_array_ref` relies on the presence of this seqdict file in some way?  Perhaps a method on that class would be more appropriate?  Something like `has_seqdict` or `has_chromosome_array_ref` or the like.